src/components/useStore/useStore.js

```diff
@@ -21,6 +21,22 @@ const useStore = create(
 				],
 			})),
 
+		editComment: id => {
+			set(state => {
+				return {
+					newBooks: state.newBooks.map(books =>
+						books.id === id
+							? {
+									...books,
+
+									edit: !books.edit,
+							  }
+							: books
+					),
```

Comment:
This would be visually more pleasing in one row. But maybe it is not totally possible due to linting.

Reply:
It sets it back after saving I guess it has to stay like that